libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/util/Geocoder.kt

```diff
@@ -0,0 +1,56 @@
+package com.mapbox.navigation.dropin.util
+
+import com.mapbox.api.geocoding.v5.GeocodingCriteria
+import com.mapbox.api.geocoding.v5.GeocodingCriteria.GeocodingTypeCriteria
+import com.mapbox.api.geocoding.v5.MapboxGeocoding
+import com.mapbox.api.geocoding.v5.models.CarmenFeature
+import com.mapbox.api.geocoding.v5.models.GeocodingResponse
+import com.mapbox.geojson.Point
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.suspendCancellableCoroutine
+import kotlinx.coroutines.withContext
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+import kotlin.coroutines.resume
+import kotlin.coroutines.resumeWithException
+
+/**
+ * Wrapper class for [MapboxGeocoding].
+ */
+internal class Geocoder(
+    private val accessToken: String,
+    private val dispatcher: CoroutineDispatcher = Dispatchers.IO
```

Comment:
The dispatcher is unnecessary, the MapboxService already makes the call on a background thread.